### DISCUSSION TITLE
a script that can import multi Ai files by just giving the names in order to import them from an specific folder

### DISCUSSION THREAD

**Discussion by raiesidesign** [Silly-V/Adobe-Illustrator#53] [Q&A]
he guys:
is there any script like that 
is that possible to make such script that instead of importing files 1 by 1 just you typefilesnames and then script import them in your file
(the files are brush, i have about 2000 brushes and its hard somm time find them and it take time , do u hv any suggestion or script that can do that ?
tnx

**Comment by Silly-V** [comment]
You can probably search in your file finder to drag in multiple files into a document?
Ask this question on the scripting forum at (https://community.adobe.com/t5/illustrator-discussions/bd-p/illustrator) and there may be some immediate help - just describe your workflow in detail on there.
